### PR TITLE
Readme zh

### PR DESCRIPTION
**Pull request by luckey01**
add chinese translation of PV readme

### CODE REVIEW

File: doc/chinese/README.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
+## 摘要
+
+这是 adjust™ 的安卓购买SDK。您可以访问[adjust.com]了解更多有关 adjust™ 的信息。
```
Comment (zhuosun) [inline doc/chinese/README.md]: `购买SDK` sounds funky... I would like to use a meaningful chinese name for it, like `收入验证SDK`

@Bingjiang13 @luckey01

File: doc/chinese/README.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        AdjustPurchase.init(config);
+    }
+}
+```
+
+![][application_config]
+
+使用您的应用识别码替换 `{YourAppToken}` 。您可以在[控制面板]找到该识别码。
+
+取决于您的应用制作是用于测试或产品开发目的，您必须将 `environment` （环境模式）设为以下值之一：
```
Comment (zhuosun) [inline doc/chinese/README.md]: `取决于您的应用制作是用于测试或产品开发目的` should be `鉴于您的应用是用于测试还是产品开发`

File: doc/chinese/README.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+```java
+String environment = ADJPConstants.ENVIRONMENT_SANDBOX;
+String environment = ADJPConstants.ENVIRONMENT_PRODUCTION;
+```
+
+**重要:** 仅当您或其他人测试您的应用时，该值应设为 `ADJPConstants.ENVIRONMENT_SANDBOX` 。在您发布应用之前，请确保将环境改设为 `ADJPConstants.ENVIRONMENT_PRODUCTION` 。再次研发和测试时，请将其设回为 `ADJPConstants.ENVIRONMENT_SANDBOX` 。
+
+我们按照设置的环境来区分真实流量和来自测试设备的测试流量。非常重要的是，您必须始终让该值保持有意义！
```
Comment (zhuosun) [inline doc/chinese/README.md]: just bad translation of `It is very important that you keep this value meaningful at all times!`

a better one would be: `所以正确使用环境参数是非常重要的！`

**Comment by luckey01** [comment]
@uerceg @nonelse this can be merged

**Pull request merged**